pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -201,19 +203,22 @@ let
       python_arch = "ia32";
     } // (extraAttrs.gypFlags or {}));
 
-    configurePhase = ''
+    preConfigure = ''
       echo "Precompiling .py files to prevent race conditions..." >&2
       python -m compileall -q -f . > /dev/null 2>&1 || : # ignore errors
 
       # This is to ensure expansion of $out.
       libExecPath="${libExecPath}"
       python build/linux/unbundle/replace_gyp_files.py ${gypFlags}
-      python build/gyp_chromium -f ninja --depth . ${gypFlags}
     '';
 
+    dontAddPrefix = true;
+    configureScript = "python build/gyp_chromium";
```

Comment:
Yep, or just call it `$gypFlags` because even `-f ninja` and `-I` are gyp flags already.